### PR TITLE
Added PSS to contrib/baseline and contrib/restricted as kustomize components

### PR DESCRIPTION
**Pull request by biswajit-9776**
# Pull Request Template for Kubeflow manifests Issues

- Please include a summary of changes and the related issue. 
- List any dependencies that are required for this change. 
- Please delete the options that are not relevant.
- The following checklist will help you to satisfy the requirements.

  
## ✅ Unit Test Checklist
  
  - [ ] 🛠️ Make sure you have installed kustomize == 5.2.1+     
  - [ ] ✍️ Have you written new tests for your core changes, as applicable?      
  - [ ] 🔄 Have you successfully run existing tests with your changes ?    
  - [ ] 🚀 Have you successfully run existing and new tests with your changes ?

## ✅ Contributor checklist
  - [ ] All the commits have been _signed-off_  (To pass the `DCO` check)
  - [ ] Submit the [Contributor License Agreements](https://cla.developers.google.com/clas) (To pass the `cla/google` check)

---     
 
> You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

### CODE REVIEW

File: contrib/security/PSS/static/baseline/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+- ../../../../../common/kubeflow-namespace/base
```
Comment (juliusvonkohout) [inline contrib/security/PSS/static/baseline/kustomization.yaml:5]: The two resources here are already in example.yaml. Why do you list them here as well?

You should add this component here commented out to example.yaml instead as planned in the Google document..

File: contrib/security/PSS/static/restricted/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+- ../../../../../common/kubeflow-namespace/base
+- ../../../../../common/istio-1-22/istio-namespace/base
```
Comment (juliusvonkohout) [inline contrib/security/PSS/static/restricted/kustomization.yaml:6]: The two resources here are already in example.yaml. Why do you list them here as well?

You should add this component here commented out to example.yaml instead as planned in the Google document.

File: example/kustomization.yaml [hunk 3]
```diff
@@ -88,3 +88,9 @@ resources:
 # KServe
 - ../contrib/kserve/kserve
 - ../contrib/kserve/models-web-app/overlays/kubeflow
+
+#Pod Security Standarards enabled
```
Comment (juliusvonkohout) [inline example/kustomization.yaml:92]: Please check the spelling

**Comment by biswajit-9776** [comment]
Should be have PSS for other namespaces like auth, cert-manager, etc?

File: example/kustomization.yaml [hunk 4]
```diff
@@ -89,7 +89,7 @@ resources:
 - ../contrib/kserve/kserve
 - ../contrib/kserve/models-web-app/overlays/kubeflow
 
-#Pod Security Standarards enabled
+#Pod Security Standards
```
Comment (juliusvonkohout) [inline example/kustomization.yaml:92]: Lets maybe link to the documentation in the comment https://kubernetes.io/docs/concepts/security/pod-security-standards/

**Pull request merged**